src/app/ui/overrides/adf-document-list.theme.scss

```diff
@@ -58,6 +58,16 @@
 
       &.adf-datatable-cell--fileSize {
         min-width: 80px;
+
+        span {
```

Comment:
Can you move this to ADF please?

Reply:
Moved ;) https://github.com/Alfresco/alfresco-ng2-components/pull/4902 and ready for review

Reply:
If it’s in the ADF with that PR, we don’t need to duplicate the styles in our code

Reply:
The extra ADF styles will be removed with [ACA-2361](https://issues.alfresco.com/jira/browse/ACA-2361) which is waiting for the upgrade to 3.4.0 ADF

Reply:
but, you are right, I'll remove the extra styles now